homeassistant/components/light/__init__.py

```diff
@@ -302,6 +318,11 @@ def color_xy(self):
         return None
 
     @property
+    def color_ct(self):
```

Comment:
I am a bit confused to what this abbreviation stands for. Does it stand for Color _ Color Temperature? If that is the case, a better name might be `color_temperature` ? To prevent confusion, this should be the same name used for the attribute in the state and the argument in the service call.